bots/hollaback.py

```diff
@@ -15,7 +15,7 @@
 PINATA_SECRET_API_KEY = os.environ.get("PINATA_SECRET_API_KEY")
 
 app = SilverbackApp()
-warper = accounts.load("warpNinjagod")
+warper = accounts.load(os.environ.get("APE_ACCOUNT_ALIAS"))
```

Comment:
You can use `app.signer` if you set `SILVERBACK_SIGNER_ALIAS` instead (and then you can delete this line as it comes for free with your silverback app)